portfolio/src/main/webapp/hobbies.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Hobbies</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+      <div class="content">
+          <h1> Hobbies </h1>
+          <nav>
+            <ul id="navigation">
+                <li><a href="index.html"> Home </a></li>
+                <li><a href="projects.html"> Projects </a></li>
+                <li><a href="hobbies.html"> Hobbies </a></li>
+            </ul>
+          </nav>
+          <div class="alt1">
+            <h2> Hiking </h2>
+            <img class="project-image" src="images/hike.jpg"/>
+            <p> I love the outdoors and hiking! Last summer, I lived in the Adirondack Mountains in NY, and 
+                got to go camping, backpacking, and mountain climbing. This picture was taken at the peak of Mount Marcy,
+                the tallest peak in the state of New York! </p>
+          </div>
+          <div class="alt2">
+            <h2> Rock Climbing </h2>
+            <img class="project-image" src="images/climb.jpg"/>
+            <p> Although I used to be afraid of heights, rock climbing is now one of my favorite hobbies. Cornell
+                has a rock wall on campus that I belong to, and love to top rope and boulder there! </p>
+           </div>
+           <div class="alt1">
+                <h2> Running </h2>
+                <img class="project-image" src="images/run.jpg"/>
+                <p> I have a love-hate relationship with running. Although sometimes my laziness gets the best of me, 
+                    I love to run outdoors, especially when it is raining. This photo was taken at the Central Park Half Marathon 
+                    in New York. I hope to run a full marathon one day! </p>
```

Comment:
Use consistent indenting (2 spaces looks like your rule here (which is great in my opinion)). There are multiple places where the indenting is off.